src/linker/Linker/WarningSuppressionWriter.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Mono.Cecil;
+
+namespace Mono.Linker
+{
+	public class WarningSuppressionWriter
+	{
+		private readonly LinkContext _context;
+		private readonly Dictionary<AssemblyNameDefinition, HashSet<(int, IMemberDefinition)>> _warnings;
```

Comment:
```suggestion
		private readonly Dictionary<AssemblyNameDefinition, HashSet<(int Code, IMemberDefinition Member)>> _warnings;
```

You can specify names for the items in the tuple - this will make the code using it easier to read.